### PR TITLE
Initialize 'Annotation' UI component

### PR DESCRIPTION
**Pull request by louisgv**
Per #149, I implemented 3 components for the annotation: The Annotation editor itself, a ConcernSlider component and a RadioButton component, all are generic and reusable. I also altered the behavior of `Nodule` component so that it is more flexible, allowing foreign editor (like the Annotation editor) to be inserted in the form of a `slot`

## Description

+ Implement Annotation component
+ Implement ConcernSlider component (generic)
+ Implement RadioButton component (generic)
+ Refactor Nodule component

## Reference to official issue

#149 

## Motivation and Context

Design doc reference here: http://concept-to-clinic.readthedocs.io/en/latest/design-doc.html#annotate

## How Has This Been Tested?

+ It seems front-end is lacking test standard/example so I will refrain from making unnecessary architectural decision (unless it is called for).
+ I tested this briefly with other component and it works ok.
+ This will heavily alter the behavior of the `Nodule` component. 

## Screenshots (if appropriate):
image

image

## CLA
- [+ ] I have signed the CLA; if other committers are in the commit history, they have signed the CLA as well

### CODE REVIEW

File: interface/frontend/src/assets/css/project.css [hunk 1]
```diff
@@ -4,4 +4,16 @@ div#version {
   top: .5rem;
   right: 1rem;
   z-index: 10000;
-}
\ No newline at end of file
+}
+
+div.container {
```
Comment (truefalse10) [inline interface/frontend/src/assets/css/project.css]: Do we really want this global style for all div.container? the class container might be used by other developers and would then produce some unexpected behavior

**Comment by lamby** [comment]
> Beside the global styles in project.css the PR looks good to me.

Same here :)

**Comment by louisgv** [comment]
@truefalse10 @lamby  thanks for the feedback. I re-scoped those style so that it only affect AnnotateAndSegment. 

However I would highly recommend a global style for this so that future development will not have to worry about the nav bar. Indeed it shouldn't be using the className `container`, as that will potentially conflict with bootstrap. Regardless, I think that all `view` should have an uniformed offset from the nav.

**Comment by lamby** [comment]
Thanks!

**Pull request merged**